packages/app/cypress/integration/components/search/SearchPage.js

```diff
@@ -41,15 +56,29 @@ describe('SearchPage', () => {
             });
         },
       });
-      cy.contains('Search');
 
-      cy.contains(results[0].document.title);
-      cy.contains(results[0].document.text);
-      cy.get(`a[href="${results[0].document.location}"]`).should('be.visible');
+      cy.expect(Heading('Search').exists());
```

Comment:
This would be closer in parity to the cypress assertion
```suggestion
      cy.expect(Heading('Search').is({ visible: true }));
```

Reply:
It doesn't matter, we check visibility implicitly